draft-ietf-rtcweb-jsep.xml

```diff
@@ -392,14 +392,13 @@ setRemote(ANSWER) |                                   |
       <section title="Session Description Format"
       anchor="sec.session-description-forma">
 
-        <t>In the WebRTC specification, session descriptions are
-        formatted as SDP messages. While this format is not optimal for
+        <t>JSEP's session descriptions use SDP syntax for their internal
+        representation. While this format is not optimal for
         manipulation from Javascript, it is widely accepted, and
         frequently updated with new features. Any alternate encoding of
         session descriptions would have to keep pace with the changes
         to SDP, at least until the time that this new encoding eclipsed
-        SDP in popularity. As a result, JSEP currently uses SDP as the
-        internal representation for its session descriptions.</t>
+        SDP in popularity. As such, SDP seems like the least bad choice.</t>
```

Comment:
Perhaps we should just remove from "Any alternate encoding of" to the end of the paragraph.